content/08.transmission.md

```diff
@@ -136,6 +136,42 @@ The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@ht
 
 ### Factors Influencing Susceptibility
 
+As COVID-19 has spread into communities around the globe, it has become clear that the risks associated with this disease are not equally shared by all individuals or all communities.
+Significant disparities in outcomes have led to interest in the demographic, biomedical, and social factors that influence COVID-19 severity.
+Untangling the factors influencing COVID-19 susceptibility is a complex undertaking.
+Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
+However, hospitalization rates vary by location [@doi:10.15585/mmwr.mm6915e3].
+This variation may be influenced by demographic (e.g., average age in the area), medical (e.g., the prevalence of comorbid conditions such as diabetes), and social (e.g., income or healthcare availability) factors that vary geographically.
+Additionally, some of the same factors may influence an individual's probability of exposure to SARS-CoV-2, their risk of developing a more serious case of COVID-19 that would require hospitalization, and their access to medical support.
+As a result, quantifying or comparing susceptibility among individuals, communities, or other groups requires consideration of a number of complex phenomena that intersect across many disciplines of research.
+
+#### Biological Risk Factors
```

Comment:
```suggestion
#### Innate Biological Risk Factors
```
What do you call age and sex, especially in relation to comorbidites and "race" (which has both a biological and social component)? I haven't been able to figure this out.

Reply:
Some people use the word "risk factors" to mean anything that is statistically predictive of an outcome (so being male or Black could be a risk factor).  Sometimes I don't like that because the colloquial meaning of risk implies direct causation, and when people use words like "risk" and "effect" they very often slip into implying that correlation equals causation, even if they are smart.  More recently, some people object to calling race a risk factor because it sounds like a biological explanation and they say that racism is the real cause of the harm.  This objection doesn't make sense if you take risk factor to simply denote a statistical predictor, but it makes a lot of sense if you take risk factor to denote a direct cause.  
Age and sex are also correlated with social and economic experiences, so I don't think that I would say that they are fundamentally different from race in that regard.  
Age, sex, and race can definitely be called important demographic factors.  I wouldn't think of comorbidities as demographic though.  
You could call them all "Patient Characteristics Associated with COVID Risk."
This raises the issue of whether to mention blood type -- I mentioned it as a statistical predictor but someone objected on the grounds that the evidence wasn't consistent enough.  More to the point, it isn't really as useful when making recommendations for policy interventions, because it isn't something that you would ordinarily know about somebody.

Reply:
I originally called all of these "subpopulations of special concern," but Dr. Greene objected to this -- I think his objection was that age and sex are probably predictive because of biological differences that can't be changed, while race is probably predictive because of social injustices that ought to be intervened on.  I don't really see why this distinction is necessary in the context of this paper, but I respect his judgment.
I guess you could have "Biological Risk Factors" and then "Socioeconomic Risk Factors."  That would make sense.  Of course, that assumes you're willing to use the word "risk factor" at all.

Reply:
As I'm thinking about this now, there are also ways where age and sex might not be strictly innate biological risk factors. In a very constrained health system, triage might lead to different levels of attention being paid to people who were perceived as the least likely to recover. I think it is probably best to treat them all as variables associated with differential outcomes.

I agree that `risk factor` is different if folks take it with a direct causal interpretation. Perhaps a short lead in to the section on risk factors that notes that some may be biological and intrinsic while others are not. Just thinking off the cuff here.

Reply:
That may have been a premature objection. I just wanted to make sure we weren't conflating factors driven by society with those driven by biology.

Reply:
I guess that if we want to be really pedantic, we shouldn't use the word "innate" for age, because age can change, and innate is Latin for "born that way."  But "intrinsic" might be okay.

Reply:
I do think it might be better to think of sex as a purely biological predictor here though -- It's true that women are more likely to do caregiving work in our society, but that wouldn't explain why *men* were at higher risk of death.  As @rando2  mentioned, age definitely does have a social dimension because if you're in a nursing home and there's an outbreak, you're in real trouble.

Reply:
In general we know that women do better with many illnesses compared to men and have higher life expectancy (seemingly across the world) and that may be related to immune system differences (due to women's immune systems being able to carry pregnancies for example; women are also more likely to get more auto-immune diseases, but the immune benefits definitely seem to outweigh this.)

Reply:
Thank you all so much for these thoughts! I really appreciated the break down of the causal implication that people may take, and I agree proactively addressing it seems wise! I tried to address the feedback in the latest commit by adding a few lines to the intro paragraph and changing the title to "Intrinsic Traits Associated with Increased Risk" (?)

Reply:
If you want, you could say "Patient Traits" instead of "Intrinsic Traits" so that all the philosophers eager to argue about exactly what is and is not intrinsic can take the day off -- It's up to you though.

Reply:
I love it @dziakj1! I definitely do not want to argue with philosophers!